### PR TITLE
Issue 90

### LINKED ISSUE #318: [90] Issue #90 - Oct 5, 2017
To contribute to this issue, simply leave a comment here. Please also review [our contributing guidelines](https://github.com/SwiftWeekly/swiftweekly.github.io/blob/master/CONTRIBUTING.md).

The current draft for this issue in [`_drafts/`](https://github.com/SwiftWeekly/swiftweekly.github.io/tree/master/_drafts). If you want to contribute directly, [open a pull request](https://github.com/SwiftWeekly/swiftweekly.github.io/compare?expand=1).

### PR DESCRIPTION
**Pull request by BasThomas**
Closes #318

### CODE REVIEW

File: _drafts/2017-10-05-issue-90.md [hunk 1]
```diff
@@ -9,52 +9,74 @@ sponsor:
     displaylink: TODO
 ---
 
-> TODO: intro comments
+It's been a while since I last wrote an issue, but I am glad to be writing another one. Especially after meeting JP at [FrenchKit](http://frenchkit.fr) where we both gave a talk (JP did one on profiling Swift performance on Linux, I did one on what we can expect from Swift 5). There will be videos, but we'll have to wait a little longer for those.
```
Comment (jessesquires) [inline _drafts/2017-10-05-issue-90.md]: link to slides for each talk?

**Comment by jessesquires** [comment]
Feel free to merge whenever! 🥇

File: _drafts/2017-10-05-issue-90.md (hunk 1 above)
Comment (jpsim) [reply _drafts/2017-10-05-issue-90.md]: JP's: https://speakerdeck.com/jpsim/performance-profiling-swift-on-linux
Bas's: https://speakerdeck.com/basthomas/whats-up-with-swift-5

**Comment by BasThomas** [comment]
https://github.com/apple/coremltools

**Pull request merged**